lib/src/components/NxTabs/types.ts

```diff
@@ -8,8 +8,8 @@ import { HTMLAttributes, ReactNode } from 'react';
 import PropTypes from 'prop-types';
 
 export type Props = HTMLAttributes<HTMLDivElement> & {
-  activeTab?: number | null ;
-  onTabSelect: ((index: number) => void);
+  activeTab?: number | null;
+  onTabSelect?: ((index: number) => void) | null;
```

Comment:
Why'd you make this optional?  I don't think it should be.

Reply:
> Why'd you make this optional? I don't think it should be.

I didn't. This was merged from your branch with the new `:focus` styles. I assumed that because it was in your branch you had a reason.

Reply:
> I didn't. This was merged from your branch with the new :focus styles. I assumed that because it was in your branch you had a reason.

:thinking: no, I don't think so.  Does it still compile without it?

Reply:
> > I didn't. This was merged from your branch with the new :focus styles. I assumed that because it was in your branch you had a reason.
> 
> 🤔 no, I don't think so. Does it still compile without it?

It does, so now it's gone again: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/e279fc6239b2fbd3b4ad9bce5219f8bb75abc358

Reply:
> It does, so now it's gone again: e279fc6

There were still other remaining changes there, mainly the `| null` part.  I've reverted it to what's on master. e2fca1cc